src/cli/cli.hpp

```diff
@@ -857,6 +858,9 @@ class Interpreter : public Output
         {"thread", &Interpreter::ProcessThread},
 #if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
         {"trel", &Interpreter::ProcessTrel},
+#endif
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+        {"tvcheck", &Interpreter::ProcessThreadVersionCheckSkip},
```

Comment:
Add documentation to [src/cli/README.md](https://github.com/openthread/openthread/blob/main/src/cli/README.md).

Reply:
Added here:
https://github.com/openthread/openthread/pull/7182/files#diff-b2c6cd8edea596db4109fafedf7588be34f777011078863009be8ca6eabc2e33R2598